src/components/SettingsManager/SharedSessionManager/SharedSessionManager.Room.tsx

```
@@ -15,26 +17,91 @@
+                  <Typography sx={{ fontWeight: "bold" }}>
+                    Next auto reconnect:{" "}
+                    <Timer onTick={getRemainingAutoReconnectTime} />
+                  </Typography>
+                  <Button
+                    onClick={() => connectToRoom(roomName)}
+                    variant="contained"
+                  >
+                    Reconnect now
+                  </Button>
+                </Box>
+              </>
```

Biome

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)